legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/project/SimpleProjectConfiguration.java

```diff
@@ -86,14 +85,10 @@ public void setProjectId(String projectId)
     }
 
     @Override
+    @JsonIgnore
```

Comment:
I'm not sure if you're okay with this @kevin-m-knight-gs

Reply:
What is it you're trying to do with the annotation?

Reply:
Addressed below